src/idfToolsManager.ts

```diff
@@ -113,6 +113,11 @@ export class IdfToolsManager {
         ) > -1
       );
     });
+    if (!versions || versions.length === 0) {
+      throw new Error(
+        `${pkg.name} doesn't have a compatible version for ${this.platformInfo.platformToUse}`
+      );
+    }
```

Comment:
Did we add `try/catch` in the caller of this function `obtainUrlInfoForPlatform(...)`, since earlier it wasn't throwing now it is.

Reply:
Hmm not in its direct caller but on the root caller.

This function is used for `downloadPackage`, `installPackages`, `installZipPackage` and `installTarPackage`. A call tree looks like

downloadToolsInIdfToolsPath
---> downloadPackages
------> downloadPackage
---> installPackages
------> installZipPackage (a or)
------> installTarPackage (b)

and downloadToolsInIdfToolsPath is called by Onboarding with a `.catch`